src/Halogen/Infinite/Scroll.purs

```diff
@@ -84,6 +88,7 @@ component =
                           , pages: Map.empty 
                           , preloaded: Map.empty
                           , lock: Nothing
+                          , update: { topLoad: Nothing, bottomLoad: Nothing, scroll: Nothing }
```

Comment:
I don't love this. It is the action to be performed after an atomic state update. It would be nice to have `H.modify` in Halogen that allows a value to be returned but since we don't have that we have this piece of state.